packages/pigeon/pigeons/core_tests.dart

```diff
@@ -200,6 +200,14 @@ abstract class HostIntegrationCoreApi {
   @SwiftFunction('echoAsync(_:)')
   String echoAsyncString(String aString);
 
+  /// Returns List of error info asynchronously.
```

Comment:
How about "Responds with an error from an async function returning a value." so it's clear on the native side what it's supposed to do.